extensions/shared/jetpack-plugin-sidebar.js

```diff
@@ -20,10 +20,6 @@ const JetpackPluginSidebar = ( { children } ) => <Fill>{ children }</Fill>;
 JetpackPluginSidebar.Slot = () => (
 	<Slot>
 		{ fills => {
-			if ( ! fills.length ) {
-				return null;
-			}
-
```

Comment:
I don't think we should remove this check, since we might otherwise end up with situations where we're displaying and empty sidebar, see e.g. https://github.com/Automattic/wp-calypso/pull/30663.

I might have an alternative idea which I'll try out shortly...

Reply:
> I might have an alternative idea which I'll try out shortly...

#12049